api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/jaxrs/MetricHandler.java

```diff
@@ -100,355 +96,239 @@
             + "a metric before storing data. Doing so however allows clients to prevent naming collisions and to "
             + "specify tags and data retention.")
     @ApiResponses(value = { @ApiResponse(code = 200, message = "Metric definition created successfully"),
-            @ApiResponse(code = 400, message = "Metric with given id already exists or request is otherwise incorrect"),
```

Comment:
The create numeric/availability metric part has been updated in #168 . I'll have to rebase the PR once this one is merged.